client/jetpack-connect/woo-loader.tsx

```diff
@@ -75,7 +75,7 @@ export const WooLoader = ( { className, stages }: Props ) => {
 	return (
 		<div className={ clsx( 'jetpack-connect-woocommerce-loader', className ) }>
 			<div className="jetpack-connect-loader-wrapper">
-				{ stage.image && stage.image }
+				<stage.image />
```

Comment:
Since the image type is `?React.ComponentType`, we can change the `Stage.image` type.

```suggestion
				{ stage.image && <stage.image /> }
```